tests/tracer/test_writer.py

```diff
@@ -176,7 +176,7 @@ def test_drop_reason_trace_too_big(self):
     def test_drop_reason_buffer_full(self):
         statsd = mock.Mock()
         writer_metrics_reset = mock.Mock()
-        writer = AgentWriter(agent_url="http://asdf:1234", buffer_size=5300, dogstatsd=statsd, report_metrics=False)
+        writer = AgentWriter(agent_url="http://asdf:1234", buffer_size=5235, dogstatsd=statsd, report_metrics=False)
```

Comment:
actually needed?

Reply:
yes, the difference is due to parent_id not being present in the encoded data.